app/controllers/orders_controller.rb

```diff
@@ -1,7 +1,11 @@
 class OrdersController < ApplicationController
 
   def index
-    @user = User.find(params[:user_id])
+		if current_user != nil
+			@user = current_user
+		else
+    	@user = User.find(params[:user_id]) || current_user
```

Comment:
Does the '|| current_user' ever evaluate or does this still run without including `|| current_user`?